tools/gen-providers.ts

```diff
@@ -1429,6 +1429,7 @@ const cloudfront_cache_behavior: RecordDecl = {
     requiredField('allowed_methods', listType(STRING)),
     requiredField('cached_methods', listType(STRING)),
     requiredField('forwarded_values', recordType(cloudfront_forwarded_values)),
+    // path_pattern cannot be set for default cache behaviour, defaults to *
     optionalField('path_pattern', STRING),
```

Comment:
Fields have an optional doco strings array:
```suggestion
    optionalField('path_pattern', STRING, ["path_pattern cannot be set for default cache behaviour, defaults to *"]),
```